roles/zuul/templates/docker-compose.yaml.j2

```diff
@@ -87,6 +87,8 @@ services:
     volumes:
       - "{{ zuul_component_conf_dirs.log_server }}:/usr/local/apache2/conf:ro"
       - "log_server:/usr/local/apache2/htdocs:z"
+      - "/etc/letsencrypt:/etc/letsencrypt:ro"
+      - "/opt/zuul/log_server/.well-known/acme-challenge/:/usr/local/apache2/htdocs/.well-known/acme-challenge/:rw"
```

Comment:
I wonder if this should also be optional depending on `zuul_use_acme`. There also should be tasks that ensure that these directories exist on the host.